examples/4337-passkeys/package.json

```
@@ -6,21 +6,22 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0 --fix",
     "preview": "vite preview"
   },
   "dependencies": {
-    "@web3modal/ethers": "^4.1.1",
+    "@web3modal/ethers": "^4.1.5",
     "@safe-global/safe-4337": "^0.3.0",
+    "@safe-global/safe-passkey": "*",
```

Should both dependencies look like this?

```suggestion
    "@safe-global/safe-4337": "*",
    "@safe-global/safe-passkey": "*",
```